apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -146,6 +165,21 @@ const CourseCalendarEvent = (props: CourseCalendarEventProps) => {
 
         const buildingId = locationIds[buildingName] ?? 69420;
 
+        let finalExamString = '';
+        if (finalExam.examStatus == 'NO_FINAL') {
+            finalExamString = 'No Final';
+        } else if (finalExam.examStatus == 'TBA_FINAL') {
+            finalExamString = 'Final TBA';
+        } else {
+            if (finalExam.startTime && finalExam.endTime && finalExam.month && finalExam.bldg) {
+                const timeString = translate24To12HourTime(finalExam.startTime, finalExam.endTime);
+                const locationString = `at ${finalExam.bldg.join(', ')}`;
+
+                // prettier-ignore
```

Comment:
why does this need a prettier-ignore?

Reply:
Prettier formats the string like so, which is not awesome

```typescript
                finalExamString = `${finalExam.dayOfWeek} ${MONTHS[finalExam.month]} ${
                    finalExam.day
                } ${timeString} ${locationString}`;
```

Reply:
I'm pretty sure it breaks on the braced property accessor, i.e. something like this
```ts
const finalExamString = `${finalExam.dayOfWeek} ${MONTHS[
  finalExam.month
]} ${ finalExam.day } ${timeString} ${locationString}`;
```

a simple workaround is just to declare a variable to hold it; prettier won't forcefully break up the variable unless it's able to do so without affecting the string, i.e. when you use braces

```ts
const finalExamMonth = MONTHS[finalExam.month]
const finalExamString = `${finalExam.dayOfWeek} ${finalExamMonth} ${finalExam.day} ${timeString} ${locationString}`;
```

Reply:
ok, agreed that looks gross 🤮

Reply:
> ```ts
> const finalExamMonth = MONTHS[finalExam.month]
> ```

Yup. That was the issue. I've pushed your suggestion :)